pkg/controller/issueapi/issue.go

```diff
@@ -30,15 +30,34 @@ import (
 func (c *Controller) HandleIssue() http.Handler {
 	logger := c.logger.Named("issueapi.HandleIssue")
 
+	type FormData struct {
```

Comment:
why this change?

I'm concerned about drift in API definition.

Reply:
test date should be added to the adminapi version as well

Reply:
I think I addressed this in 237483a